pkg/cmd/docgen/diagrams.go

```diff
@@ -575,6 +575,29 @@ var specs = []stmtSpec{
 		replace: map[string]string{"relation_expr": "view_name", "'RENAME' 'TO' view_name": "'RENAME' 'TO' view_new_name"},
 		unlink:  []string{"view_name", "view_new_name"},
 	},
+	{
+		name:    "alter_virtual_cluster",
+		stmt:    "alter_virtual_cluster_stmt",
+		inline:  []string{"virtual_cluster", "replication_options_list", "alter_virtual_cluster_capability_stmt", "alter_virtual_cluster_replication_stmt", "alter_virtual_cluster_rename_stmt", "alter_virtual_cluster_service_stmt"},
+		replace: map[string]string{"'ON' d_expr": "'ON' physical_cluster", "d_expr": "virtual_cluster_spec", "a_expr": "timestamp"},
+		unlink:  []string{"physical_cluster", "virtual_cluster_spec", "timestamp"},
+	},
+	{
```

Comment:
I'm unclear on why the following 4 entries are necessary, unless you intend to use the separate generated BNF files elsewhere. If you inlined them in the `alter_virtual_cluster` definition, shouldn't they all appear under that single diagram (as you showed in the screenshot)?

Reply:
hmmmm, it seemed like they were necessary when I used `dev generate bnf`... that is, I followed the error messages and added the required stmt entries. I can take another look if you've seen that these aren't necessary.

Reply:
If you followed the output then I think that's fine!

Reply:
Just to note here as well, I followed step 4 and the content looked good 👍 Thanks!